lib/core/src/Cardano/Wallet/Primitive/AddressDiscovery/Sequential.hs

```diff
@@ -152,7 +152,7 @@ instance ToText (AddressPoolGap) where
 
 instance Bounded AddressPoolGap where
     minBound = AddressPoolGap 10
-    maxBound = AddressPoolGap 100
+    maxBound = AddressPoolGap 100000
```

Comment:
Just a suggestion, but I find these long constants easier to read with `NumericUnderscores`, so:

```suggestion
    maxBound = AddressPoolGap 100_000
```
Perhaps my eyes are getting old :smile_cat:

Reply:
Done :+1: , I thought I did it already :shrug: ... I thought of it for sure.